sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -157,22 +157,16 @@ static Sampler configureSampler(
         return Sampler.alwaysOff();
       case "traceidratio":
         {
-          Double ratio = config.getDouble("otel.traces.sampler.arg");
-          if (ratio == null) {
-            ratio = 1.0d;
-          }
+          double ratio = config.getDouble("otel.traces.sampler.arg", DEFAULT_SAMPLE_RATIO);
           return Sampler.traceIdRatioBased(ratio);
         }
-      case "parentbased_always_on":
+      case DEFAULT_SAMPLER:
```

Comment:
Whether `parentbased_always_on` is the default or not isn't important here, let's stick with the string since we're just parsing strings in this switch statement

Reply:
Ah ok, the goal was to eliminate having the string constant in two places. I'll rename it to reflect the contents instead.